src/transactions/BumpSequenceOpFrame.cpp

```diff
@@ -0,0 +1,102 @@
+// Copyright 2017 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#include "transactions/BumpSequenceOpFrame.h"
+#include "transactions/TransactionFrame.h"
+#include "crypto/SignerKey.h"
+#include "database/Database.h"
+#include "main/Application.h"
+#include "medida/meter.h"
+#include "medida/metrics_registry.h"
+
+namespace stellar
+{
+using xdr::operator==;
+
+
+BumpSequenceOpFrame::BumpSequenceOpFrame(Operation const& op, OperationResult& res,
+                                     TransactionFrame& parentTx)
+    : OperationFrame(op, res, parentTx)
+    , mBumpSequence(mOperation.body.bumpSequenceOp())
+{
+}
+
+ThresholdLevel
+BumpSequenceOpFrame::getThresholdLevel() const
+{
+    // bumping sequence is low threshold
+    return ThresholdLevel::LOW;
+}
+
+bool
+BumpSequenceOpFrame::doApply(Application& app, LedgerDelta& delta,
+                           LedgerManager& ledgerManager)
+{
+    Database& db = ledgerManager.getDatabase();
+    // Account must be loaded from loadAccount to ensure data is fresh & account not deleted
+    AccountFrame::pointer bumpAccount =
+        AccountFrame::loadAccount(delta, mSourceAccount->getID(), db);
+    SequenceNumber current = bumpAccount->getSeqNum();
+
+    // fail if the account couldn't be found
+    if (!bumpAccount)
```

Comment:
I think this will be catched much earlier: https://github.com/stellar/stellar-core/blob/master/src/transactions/OperationFrame.cpp#L150 with opNO_ACCOUNT error, as it is called by https://github.com/stellar/stellar-core/blob/master/src/transactions/OperationFrame.cpp#L97